src/main/java/net/dries007/tfc/config/ServerConfig.java

```diff
@@ -359,6 +378,43 @@
 
         hotWaterHealAmount = builder.apply("hotWaterHealAmount").comment("An amount that sitting in hot water will restore health, approximately twice per second.").defineInRange("hotWaterHealAmount", 0.08, 0.0, 20.0);
 
+        innerBuilder.pop().push("saplings");
+
+        globalSaplingGrowthModifier = builder.apply("globalSaplingGrowthModifier").comment("Modifier applied to the growth time of every (non-fruit) sapling").defineInRange("globalSaplingGrowthModifier", 1d, 0d, Double.MAX_VALUE);
+        globalFruitSaplingGrowthModifier = builder.apply("globalFruitSaplingGrowthModifier").comment("Modifier applied to the growth time of every fruit tree sapling").defineInRange("globalFruitSaplingGrowthModifier", 1d, 0d, Double.MAX_VALUE);
```

Comment:
Neither of these specify if it gets larger or shorter based on modifier changes.